src/main/java/webApp/SurveyMessage.java

```diff
@@ -7,17 +7,39 @@
  */
 public class SurveyMessage {
 
+    private final Integer id;
     private final String name;
+    private final String status;
     private final Collection<QuestionMessage> questions;
 
     /*
      * A constructor
+     * @param id {Integer}
+     * @param status {String}
      * @param name {String}
      * @param questions {Collection<QuestionMessage>}
      */
-    public SurveyMessage(String name, Collection<QuestionMessage> questions) {
+    public SurveyMessage(Integer id, String status, String name, Collection<QuestionMessage> questions) {
```

Comment:
Can we make status an optional parameter, if we choose to include it or not when sending a survey message

Reply:
If the JSON is missing the status then it will be automatically filled in. So what you are asking for is the case.